src/app.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <link rel="icon" href="%sveltekit.assets%/icon.svg" />
+  <meta name="viewport" content="width=device-width" />
+
+  %sveltekit.head%
+</head>
+
+<body data-sveltekit-preload-data="hover">
```

Comment:
what are the other options? What does it mean?

Reply:
This tells SvelteKit to try to preload the next page when hovering a link in anticipation of a click, and is the default. The other option is "tap" which doesn't do this.